modules/ROOT/pages/manage-secure-properties.adoc

```diff
@@ -40,7 +40,9 @@ sudo ./rtfctl get secure-properties -n <environment_id>
 
 == Access Secure Properties
 
-You can access secure properties from inside your Mule application using `${my.property.name}`.
+You can access secure properties from inside your Mule application using `$key`, which is the key provided in the `rtfctl apply` command. You can retrieve the key just like any other xref:mule-runtime/configuration-properties.adoc[configuration property].
+
```

Comment:
I'd suggest a slight change to the second sentence:
You can retrieve the key the same way you retrieve any other ...